build.js

```diff
@@ -539,7 +546,12 @@ const buildIDLTests = (ast) => {
   const tests = {};
 
   const interfaces = ast.filter((dfn) => {
-    return dfn.type === 'interface' || dfn.type === 'namespace';
+    return (
+      dfn.type === 'interface' ||
+      dfn.type === 'namespace' ||
+      // WindowOrWorkerGlobalScope is mapped differently in BCD
```

Comment:
Rather than adding the `WindowOrWorkerGlobalScope` definition to `interfaces` here, how about doing `const globals = ast.find((dfn) => dfn.name === 'WindowOrWorkerGlobalScope')` and dealing with `globals` before or after `interfaces`? That should avoid a lot of special casing inside the `for (const iface of interfaces)` loop, and there really isn't a lot of code in common to share. The generated test would just be `'fetch' in self` and similar.